components/site-search/server.css

```diff
@@ -6,8 +6,19 @@
   @media (--screen-layout-no-sidebar) {
     grid-template-columns: var(--layout-no-sidebar);
   }
-}
 
-mdn-site-search {
-  grid-column: content;
+  h1,
+  mdn-site-search {
+    grid-column: content;
+  }
+
+  /* Same as: `.not-found h1` */
+  h1 {
+    margin-top: 0;
+
+    font-size: var(--font-size-largest);
+    font-weight: normal;
+
+    line-height: var(--font-line-content);
```

Comment:
This looks very large, costing a lot of screen estate. Maybe cut back on the font size / line height? My understanding is that this is primarily an a11y / semantic issue, visually this is not really needed to use the page.

Putting it in front of the input field would also be a possibility, minimizing the impact on vertical space, but then the wording should be slightly different: "Search for [input field]", assuming that would work in all languages.

Reply:
Google seems to have a visually hidden `<h1>` on their results page, I wonder if we should take the same approach:

<img width="1236" height="282" alt="image" src="https://github.com/user-attachments/assets/bfc23f95-c98a-4352-90d4-13b05f540ee3" />